lib/engine/game/g_18_norway/steps/build_tunnel.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G18Norway
+      module Step
+        class BuildTunnel < Engine::Step::Base
+          def description
+            'Build tunnel'
+          end
+
+          def actions(entity)
+            hex = @round.mountain_hex
```

Comment:
Lots of methods in this file are repeating this line `hex = @round.mountain_hex`. You could make things simpler by defining a private method `hex` that returns `@round.mountain_hex`, similar to [here](https://github.com/tobymao/18xx/blob/4a83cc6eb87495f33c8f4ae967be81b99a17606a/lib/engine/step/base.rb#L108-L112).

Reply:
Refactored